packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/CustomTabsActivitiesHelper.kt

```diff
@@ -0,0 +1,27 @@
+package expo.modules.webbrowser
+
+import expo.modules.core.interfaces.InternalModule
+import expo.modules.webbrowser.error.PackageManagerNotFoundException
+import expo.modules.core.errors.CurrentActivityNotFoundException
+import android.content.Intent
+import java.util.ArrayList
+
+interface CustomTabsActivitiesHelper : InternalModule {
+  @get:Throws(PackageManagerNotFoundException::class, CurrentActivityNotFoundException::class)
```

Comment:
Do we have a policy about annotating exceptions types? 🤔 
I'm not sure what is the standard for Kotlin, but it seems as a good approach, let's add it to our codestyle guide, shall we?